src/player.js

```diff
@@ -1045,6 +1051,25 @@ export default class Player extends FakeEventTarget {
   }
 
   /**
+   * Handles the cue text removal issue, when seeking to a time without captions in IE \ edge the previous captions
+   * are not removed
+   * @returns {void}
+   * @private
+   */
+_removeCueTextPatch(): void {
+  const textCue = this._activeTextCues;
+  if (textCue.length > 0) {
```

Comment:
`_activeTextCues` can be an array so need to iterate over all of it and throw out the out of bound cues and keep the ones in it